<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>GISAXS simulation Siegen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widgetPlot" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Beta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="beta1SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="gamma1SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>delta_t</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="deltaT1SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>n</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="n1spinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Beta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="beta2SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="gamma2SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>delta_t</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="deltaT2SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>n</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="n2spinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Beta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="beta3SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="gamma3SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>delta_t</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="deltaT3SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>n</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="n3spinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Beta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="beta4SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="gamma4SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>delta_t</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="deltaT4SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>n</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="n4spinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Beta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="beta5SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="gamma5SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>delta_t</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="deltaT5SpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>n</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="n5spinBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="randomPushButton">
        <property name="text">
         <string>Random data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusss">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menusss"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
